src/status_im/contexts/onboarding/create_profile/view.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im.contexts.onboarding.create-profile.view
```

Comment:
Refactored the [create-or-sync-profile](https://github.com/status-im/status-mobile/blob/develop/src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs) screen and split it into two separate screens:
('Create Profile' and 'Log In - as per figma)